.travis.yml

```diff
@@ -266,6 +268,12 @@ script:
     ccache --zero-stats;
 
   - echo --- Set cmake parameters;
+    COMPILE_OPT+=(
+      -DCMAKE_C_FLAGS_DEBUG=-g1
+      -DCMAKE_CXX_FLAGS_DEBUG=-g1
+      '-DCMAKE_C_FLAGS_RELWITHDEBINFO=-O2 -g1 -DNDEBUG'
+      '-DCMAKE_CXX_FLAGS_RELWITHDEBINFO=-O2 -g1 -DNDEBUG'
```

Comment:
Why do you add NDEBUG and -O2 here? https://github.com/percona/percona-server/blob/8.0/CMakeLists.txt#L937 seems to prepend NDEBUG even if you don't speficy this variable, and the O2 isn't mentioned in the commit message at all. You changed it to reduce the running time of the optimizer? If yes, then it should probably be also mentioned in the commit message.

Reply:
Look at https://github.com/percona/percona-server/blob/8.0/CMakeLists.txt#L1571
`-O2 -g -DNDEBUG` are defaults for `CMAKE_CXX_FLAGS_RELWITHDEBINFO`.
We only switch from `-g` (which translates to `-g2`) to `-g1` to decrease output size (since we have 500 MB limit for ccache for Travis).

Reply:
You are right, I checked an 5.7 release build and assumed it's the same... (I wonder why did we switch to -O2 with 8.0?)

Reply:
But I also verified the other part of my comment: `-DCMAKE_CXX_FLAGS_RELWITHDEBINFO=-g1'` is enough, `-O2` and `-NDEBUG` will be added to the build commands even with this.

Reply:
We have Release with `-O3` i.e. `-DCMAKE_CXX_FLAGS_RELEASE=-O3 -DNDEBUG`.